### PR TITLE
integrated quartermaster API

### LINKED ISSUE #32: use quartermaster APIs from infiniboard-app
* remove mock data
* implement HTTP calls to fetch data from quartermaster

resources:
* https://angular.io/docs/ts/latest/guide/server-communication.html

### PR DESCRIPTION
**Pull request by matthiasbalke**
Features
* removed mock data from `infiniboard-app`
* fetching data from quartermaster API (closes #32)

### CODE REVIEW

**Comment by matthiasbalke** [comment]
As harvester does not run any jobs ATM widget data retrieval can not be tested.
`Dashboard`s and `WidgetConfiguration`s can be fetched and displayed.

**Comment by matthiasbalke** [comment]
ready for merge

**Comment by matthiasbalke** [comment]
ready for merge

**Comment by matthiasbalke** [comment]
ready for merge. processing all widget data now.

**Comment by matthiasbalke** [comment]
added an example widget configuration (`docs/examples`)

File: doc/examples/platformVersion/widget.platform-dev.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "title": "dev",
+  "type": "platform-status",
+  "sourceConfigs": [
+    {
+      "id": "version",
+      "type": "urlSource",
+      "interval": 10000,
+      "configData": {
+        "url": "https://putsreq.com/eT3fqDqHnnNKureYOuOf"
+      }
```
Comment (Pentadrago) [inline doc/examples/platformVersion/widget.platform-dev.json]: putsreg was new to me, nice service

**Comment by matthiasbalke** [comment]
merged upstream changes. @Pentadrago or @RudiKlassen: please merge

**Pull request merged**